packages/cyberstorm/src/components/Tooltip/Tooltip.module.css

```diff
@@ -0,0 +1,87 @@
+.root {
+  max-width: 75vw;
+
+  padding: var(--space--8) var(--space--12);
+  border-radius: var(--border-radius--8);
+  color: var(--color-text--default);
+
+  font-weight: 700;
+  font-size: var(--font-size--14);
```

Comment:
I'd also add `font: var(--font-body);` here, as we pass straight up strings in the story and they look yucky.